lib/rack/tracker/facebook_pixel/facebook_pixel.rb

```diff
@@ -1,6 +1,16 @@
 class Rack::Tracker::FacebookPixel < Rack::Tracker::Handler
   self.position = :body
 
+  def tracker_options
+    @_tracker_options ||= {}.tap do |tracker_options|
+      options.each do |key, value|
+        if option_value = value.respond_to?(:call) ? value.call(env) : value
+          tracker_options[key] = option_value
+        end
+      end
+    end
+  end
```

Comment:
a similar method is in https://github.com/railslove/rack-tracker/pull/97/files#diff-8c96bad81146eddcac1965e5645e68f3R11

I think it would be awesome to move it to the Rack::Tracker::Handler class that all the trackers can access it. 
What do you think? could you do that?

Reply:
Sure, I'll give it a shot!

Reply:
yay, that would be a great addition.